client/src/css/ClassCard.scss

```diff
@@ -0,0 +1,4 @@
+.card-content-container {
+    display: flex;
+    justify-content: space-between;
+}
```

Comment:
Add a new line here. It's formatting standards to have a new line at the end of each file